pyvisa_py/protocols/rpc.py

```diff
@@ -979,8 +994,13 @@ def loop(self):
         while 1:
             self.session()
 
+    def sock_sendto(self, data, address):
```

Comment:
I would prefer to avoid duplicating this. Could you refactor this as a function taking the socket as first argument ?

Reply:
I agree, it's cleaner that way.  I'd feel more comfortable if I had a way to test the code that uses sock.sendto(), but I don't even see a way for the UDP code to get executed.  Maybe it should just be deleted?

I recall running into some device in the past that supported UDP connections, but I don't recall which instrument it was.  Most instruments I've worked with only support TCPIP connections.

Reply:
When pyvisa-py was started (before I got involved), the rpc file was pulled in as a basic implementation of RPC upon which VXI-11 is based. But it is true that large swabs of it are absolutely useless since VXI -11 does not requires a complete RPC implementation. However due to the little time I have to work on this and the difficulty to test things to avoid regression I never deleted the useless bits of that file.